test/parallel/test-crypto-authenticated.js

```diff
@@ -389,7 +389,7 @@ for (const test of TEST_CASES) {
       assert.strictEqual(msg, test.plain);
     } else {
       // assert that final throws if input data could not be verified!
-      assert.throws(function() { decrypt.final('ascii'); }, errMessages.auth);
+      assert.throws(() => decrypt.final('hex'), errMessages.auth);
```

Comment:
Micro-nit: I'd prefer we keep braces around this arrow function because it makes it clear that we're not intending to return a value here. Totally ignore me if you have strong opinions to the contrary.

Reply:
Done, thank you.